core/src/smartcontracts/wasm.rs

```diff
@@ -316,7 +362,7 @@ impl<'wrld, W: WorldTrait> Runtime<'wrld, W> {
 
         let res_offset = {
             let res_offset = alloc_fn
-                .call(&mut caller, res_bytes_len)
+                .call(&mut caller, (res_bytes_len, 1))
```

Comment:
Is there any possibility that the byte alignment will take other than `1` somewhere else?
In other words, is the reason why the byte alignment can be specified in `iroha_core_wasm::alloc::_iroha_wasm_alloc` simply for consistency with the `GlobalAlloc` trait implementation, or is there another purpose?

Reply:
Host and iroha module will exchange data serialized in scale format as a slice of bytes. Therefore, I don't think host would use alignment other than 1 and this is just for consistency